trappy/base.py

```diff
@@ -98,7 +98,8 @@ class Base(object):
     This class acts as a base class for all TRAPpy events
 
     """
-    def __init__(self, parse_raw=False):
+    def __init__(self, parse_raw=False, is_backup=False):
```

Comment:
Can you add to the documentation what `is_backup` actually means? Let's call it `is_fallback` as discussed.